cypress/support/helpers/checkA11y.js

```diff
@@ -58,6 +58,14 @@ const checkA11y = () => {
         id: 'aria-hidden-focus',
         enabled: false,
       },
+      {
+        /*
+         * Our ads containers sometimes cause a color contrast violation depending on what ad is loaded
+         * Disabling this rule for now as we cannot work out which class/element to exclude, and we carry out contrast checks in a11y swarms so this rule is not really needed
```

Comment:
Can we not make this more specific to just adverts please?

Reply:
There should be an advert class you can use?

Reply:
We did try that in the last [PR](https://github.com/bbc/simorgh/pull/9597/files). However from discussing with @jroebu14, the exclusion seems to apply on first page load, and because the ad is injected after the fact, the exclusion list doesn't work causing the test to fail. We also discussed, as we do colour contrast checks as part of a11y swarms we should be covered in that regard

Reply:
Ok... thanks @HarveyPeachey